packages/react-vapor/src/components/collapsible/CollapsibleContainerConnected.tsx

```diff
@@ -3,24 +3,110 @@ import {findWhere} from 'underscore';
 
 import {IReactVaporState} from '../../ReactVapor';
 import {ReduxUtils} from '../../utils/ReduxUtils';
-import {
-    CollapsibleContainer,
-    ICollapsibleContainerOwnProps,
-    ICollapsibleContainerProps,
-    ICollapsibleContainerStateProps,
-} from './CollapsibleContainer';
-
-const mapStateToProps = (
-    state: IReactVaporState,
-    ownProps: ICollapsibleContainerOwnProps
-): ICollapsibleContainerStateProps => {
+
+import classNames from 'classnames';
+import * as React from 'react';
+
+import {IClassName} from '../../utils/ClassNameUtils';
+import {LinkSvg} from '../svg/LinkSvg';
+import {Svg} from '../svg/Svg';
+import {ITooltipProps, Tooltip} from '../tooltip/Tooltip';
+import {CollapsibleConnected} from './CollapsibleConnected';
+import * as styles from './styles/CollapsibleContainer.scss';
+
+export interface ICollapsibleContainerOwnProps {
+    id: string;
+    title: React.ReactNode;
+    expandedOnMount?: boolean;
+    informationUrl?: string;
+    informationTooltip?: ITooltipProps;
+    className?: IClassName;
+    collapsibleHeaderClassName?: IClassName;
+    collapsibleBodyClassName?: IClassName;
+    withoutContentPadding?: boolean;
+}
+
+const mapStateToProps = (state: IReactVaporState, ownProps: ICollapsibleContainerOwnProps) => {
     const collapsibleState = findWhere(state.collapsibles, {id: ownProps.id});
 
     return {expanded: collapsibleState && collapsibleState.expanded};
 };
 
-export const CollapsibleContainerConnected: React.ComponentClass<ICollapsibleContainerProps> = connect(
+const getCollapsibleHeaderIcon = ({

```

Comment:
Convert to function component?
```suggestion
const CollapsibleHeaderIcon: React.FunctionComponent<{
    informationTooltip: ITooltipProps;
    informationUrl: string;
}> = ({informationTooltip, informationUrl}) =>
```